etc/ContributionIdeas.md

```diff
@@ -0,0 +1,40 @@
+## Planned Features
```

Comment:
Should this be in etc/ or doc/?

Reply:
I guess this can be in `etc`, `doc` is for real documentation, this is more of a placeholder for people to coordiante, maybe should even be in the wiki not in the repos.